client/txn.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Andrei Matei (andrei@cockroachlabs.com)
+
+// Package client provides helpers for using CockroachDB in client
+// applications.
+package client
```

Comment:
> There is no requirement that the package name and the directory name match.

That's true, but it does confuse `godef` (unless you manually name the import) and thus reduces tooling support in some editors (notably `vim`), which is also unfortunate.